db_router.py

```diff
@@ -0,0 +1,34 @@
+
+
+class TmpBaseRouter(object):
```

Comment:
I think we should find a better name here. `tmp` is connoted to something like `data that should be deleted at some point`, while we try to express something like `data that is not backuped`.
Maybe something around `volatil` or just `secondary`.
Let's brainstorm ;)